jujugui/static/gui/src/test/test_web_sandbox.js

```diff
@@ -50,7 +50,7 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
     describe('sendPostRequest', function() {
 
       it('uses the given state to handle local charm uploads', function() {
-        var path = '/juju-core/charms?series=trusty';
+        var path = '/juju-core/model/some-uuid/charms?series=trusty';
```

Comment:
We've run into a problem before where test data wasn't accurate enough to real world data. Consequently there was a bug that a test would have exposed, had it had more realistic data. I think anytime we're testing code that parses a string into parts, that string needs to be as true to real life as possible; hence, I'd like to see a real UUID in here.